### ISSUE TITLE
peewee field with choices

### ISSUE THREAD

**Issue by art1415926535** [insolite/graphene-peewee-async#5]
Peewee field with a `choices` argument is not passed to GraphQL.

For example we can modify [model](https://github.com/insolite/graphene-peewee-async/blob/master/tests/common/models.py#L24) from tests:
```python
GENRES = (
    (0, 'Science fiction'),
    (1, 'Guide'),
    (2, 'Horror'),
)

class Book(BaseModel):
    name = CharField()
    year = IntegerField()
    author = ForeignKeyField(Author)
    genre = IntegerField(choices=GENRES)
```
and modify [test](https://github.com/insolite/graphene-peewee-async/blob/master/tests/test_api/test_query.py#L10):
```python
        book = self.loop.run_until_complete(
            self.manager.create(Book, name='bar', year=2000, author=author, genre=1)  # genre was added
        )

        result = self.loop.run_until_complete(self.query('''
            query {
                book (id: ''' + str(book.id) + ''') {
                    id
                    name
                    year
                    genre  # this
                    author {
                        id
                        name
                        rating
                    }
                }
            }
        '''))
```

unittest:
`AssertionError: [GraphQLError('Cannot query field "genre" on type "Book".',)] is not None`

I tried to figure point of this error and found what [function](https://github.com/graphql-python/graphene/blob/f1624af08aec4344b365094951aefc9979bced07/graphene/types/utils.py#L12) from graphene return None.

So. What's wrong with my code?

**Comment by slash-oleh** [comment]
Thanks for reporting. That's definetely a bug in peewee=>graphl field conversion. I will fix it soon.

**Comment by slash-oleh** [comment]
Well, it was intended to work through `Enum` but had a [little bug](https://github.com/insolite/graphene-peewee-async/commit/d657be37a8d346745e5103cfed1d81a76729f889#diff-9d849a284ae96db5528dcee9b8356992R37) in field creation. Having that fixed I realized that using of `Enum` adds some restrictions and non-obvious value conversions as a result. If you look at the [graphene-django](https://github.com/graphql-python/graphene-django) implementation there are [uppercasing of strings](https://github.com/graphql-python/graphene-django/blob/v2.0.0/graphene_django/converter.py#L19) and [prefixing of integers](https://github.com/graphql-python/graphene-django/blob/v2.0.0/graphene_django/converter.py#L23).
And there are some discussions on this theme:
graphql-python/graphene-django#67
graphql-python/graphene-django#133
graphql-python/graphene-django#280
I'm not sure it should be this way so looking back at the issues feedback above I'd rather ignore choices at the moment and just fix the missing field.

**Comment by slash-oleh** [comment]
Released in [v2.1.3](https://github.com/insolite/graphene-peewee-async/releases/tag/v2.1.3)
